pagopa/io-app
# chore: [IOBP-423] Idpay api definitions update for barcode

## Short description
 updated idpay api definitions for barcode handling

## List of changes proposed in this pull request
- updated definitions
- rearranged barcode logic in order to follow the new api logic

## How to test
With the dev-server, point to https://github.com/pagopa/io-dev-api-server/pull/326 and make sure the barcode flow does not break.
keep in mind that the initiative name and max value are not contextual.
